data/publiccloud/terraform/gce.tf

```diff
@@ -76,6 +84,21 @@ resource "google_compute_instance" "openqa" {
 
 }
 
+resource "google_compute_attached_disk" "default" {
+  count    =  "${var.create-extra-disk ? var.instance_count: 0}"
+  disk     = "${element(google_compute_disk.default.*.self_link, count.index)}"
+  instance = "${element(google_compute_instance.openqa.*.self_link, count.index)}"
+}
+
+resource "google_compute_disk" "default" {
```

Comment:
I had to use labels instead of metadata as we do for "google_compute_instance" as metadata didn't work: http://fromm.arch.suse.de/tests/6682/file/serial_terminal.txt

And I had to skip openqa_created_date cause wrong string formation: http://fromm.arch.suse.de/tests/6681/file/serial_terminal.txt

I keep this under investigation to see what is the best way to have labels/tags in GCE. Maybe for another PR.

Reply:
ok